pkg/skaffold/deploy/status_check.go

```diff
@@ -41,9 +45,21 @@ var (
 
 	// For testing
 	executeRolloutStatus = getRollOutStatus
+
+	TabHeader = " -"
 )
 
-func StatusCheck(ctx context.Context, defaultLabeller *DefaultLabeller, runCtx *runcontext.RunContext) error {
+type Checker struct {
+	context       context.Context
+	runCtx        *runcontext.RunContext
```

Comment:
Why do we need run context here? In general we shouldn't pass runcontext around too much beyond the runner if we can avoid it.